### PR TITLE
Change application layout

### PR DESCRIPTION
**Pull request by Ukhanskyi**
[Trello Task](https://trello.com/c/2wHYAQPV/58-change-application-layout)

**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

image

### CODE REVIEW

**Review by matsevich — APPROVED** [review]

**Review by DenisRem — APPROVED** [review]

**Review by Axedez — APPROVED** [review]

File: app/javascript/stylesheets/about.scss [hunk 1]
```diff
@@ -48,23 +47,20 @@
   }
 }
 
-.profile_container h2{
+.profile-container h2{
```
Comment (pereslop) [inline app/javascript/stylesheets/about.scss:50]: Add spaces before brackets

File: app/javascript/stylesheets/basic.scss [hunk 2]
```diff
@@ -1,49 +1,76 @@
...
 }
 
 .dropdown-lang {
-  background: #798db6;
+  background-color: rgba(255, 255, 255, 0.5);
   width: 155px;
+
+  &:hover, .layout-dropdown-item:hover {
+    font-weight: bold;
+    background-color: rgba(255, 255, 255, 0.5);
```
Comment (pereslop) [inline app/javascript/stylesheets/basic.scss:61]: rgba(255, 255, 255, 0.5) used in a lot of places

File: app/views/home/about.html.slim [hunk 3]
```diff
@@ -1,5 +1,6 @@
-h1.about-title #{t('about.header')}
-p.about-paragraph #{t('about.paragraph1')} 'Ruby on Rails' <span>SoftServe</span>#{t('about.paragraph2')}
-.row
-  - AboutHelper::TEAM_MEMBERS.each do |hash|
-    = render partial: 'shared/about_profile', locals: {name: hash[:name], mail: hash[:email]}
+.layout-first-content.rounded
+  h1.about-title = t('about.header')
+  p.about-paragraph #{t('about.paragraph1')} 'Ruby on Rails' <span>SoftServe</span>#{t('about.paragraph2')}
```
Comment (pereslop) [inline app/views/home/about.html.slim:3]: No need interpolation

File: app/views/home/index.html.slim [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
+.container.w-auto.rounded.shadow.layout-content
+  .row.mt-2.rounded.layout-first-content.pt-3.pb-3
+    .layout-text.col-lg-7.mx-auto.my-auto
+      h1.strong.text-center.pb-4
+        strong
+          = t('home.title')
+      p.lead.mt-3 #{t('home.osbb_app_goal')}
+    .layout-img.col-lg-4.mx-auto.my-auto
+      .container.row.justify-content-center
+        .building-img = image_tag 'building.jpg', class: 'rounded shadow w-100 m-3', alt: "#{t('home.alt_text')}"
```
Comment (pereslop) [inline app/views/home/index.html.slim:10]: Interpolation

File: app/views/shared/_header.html.slim [hunk 5]
```diff
@@ -1,31 +1,26 @@
-header.header.w-100.mb-2.pb-2
-  nav.navbar.navbar-expand-lg.navbar-dark.navbar-header
-    = link_to home_path, class: 'nav-name navbar-brand text-black'
-      = t('header.nav_name')
+header.header.w-100.shadow.rounded.transparent
+  nav.navbar.navbar-expand-lg.navbar-header.transparent
+    = link_to home_path, class: 'nav-name navbar-brand'
+      .d-inline = image_tag 'logo2.svg', class: 'my-auto header-logo navbar-brand ml-2', alt: "#{t('home.alt_text')}"
```
Comment (pereslop) [inline app/views/shared/_header.html.slim:4]: Interpolation

File: app/javascript/stylesheets/about.scss (hunk 1 above)
Comment (Ukhanskyi) [reply app/javascript/stylesheets/about.scss:50]: done

File: app/javascript/stylesheets/basic.scss (hunk 2 above)
Comment (Ukhanskyi) [reply app/javascript/stylesheets/basic.scss:61]: Fixed )

File: app/views/home/about.html.slim (hunk 3 above)
Comment (Ukhanskyi) [reply app/views/home/about.html.slim:3]: done

File: app/views/home/index.html.slim (hunk 4 above)
Comment (Ukhanskyi) [reply app/views/home/index.html.slim:10]: done too

File: app/views/shared/_header.html.slim (hunk 5 above)
Comment (Ukhanskyi) [reply app/views/shared/_header.html.slim:4]: done

**Review by IvanDanyliv — APPROVED** [review]

**Comment by Ukhanskyi** [comment]
Full screenshot

image

**Review by Andrii-Kozak — APPROVED** [review]

File: app/javascript/stylesheets/about.scss [hunk 6]
```diff
@@ -2,7 +2,9 @@
 
 .about-title {
   font-weight: bold;
+  padding-top: 30px;
   margin: 30px;
+  margin-bottom: 10;
```
Comment (mmarusyk) [inline app/javascript/stylesheets/about.scss:7]: Use sizing

Comment (mmarusyk) [reply app/javascript/stylesheets/about.scss:7]: you can write margin: 30px 30px 10px;

**Review by pereslop — APPROVED** [review]

**Review by chu4 — APPROVED** [review]

**Review by vikdotdev — APPROVED** [review]

**Pull request merged**